app/models/experimentation/experiment.rb

```diff
@@ -15,9 +15,10 @@ class Experiment < ActiveRecord::Base
       live: "live",
       complete: "complete"
     }, _prefix: true
+
     enum experiment_type: {
-      current_patient_reminder: "current_patient_reminder",
-      inactive_patient_reminder: "inactive_patient_reminder"
+      active_patients: "active_patients",
+      stale_patients: "stale_patients"
```

Comment:
I may be misremembering, but I had previously thought you didn't like the name stale and wanted "inactive" so I changed "stale_patient_reminder" to "inactive_patient_reminder". I initially went with "current" and "stale" because the experiment had an "active" state and I wanted to avoid overloading of word.

Reply:
Good point, lemme push up a refinement on this.